IRA 1.0.0 TI Prepub Edits

MaryLJ opened the pull request:
<!-- 
Thanks for creating this pull request 

Please make sure that the pull request is limited to one Issue and keep it as small as possible. You can open multiple pull request instead of opening a huge one.
-->

<!-- If this pull request closes an issue, please mention the issue number below -->
Closes # <!-- Issue # here -->

## 📑 Description
<!-- Add a brief description of the pr -->
Radiology IRA Rev. 1.0.0 (TI) prepublication edits
<!-- You can also choose to add a list of changes and if they have been completed or not by using the markdown to-do list syntax
- [ ] Not Completed
- [x] Completed
-->

## ✅ Checks
<!-- Make sure your pr passes the CI checks and do check the following fields as needed - -->
- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed
- [x] I have selected a committee co-chair to review the PR

## ℹ Additional Information
<!-- Any additional information like breaking changes, dependencies added, screenshots, comparisons between new and old behavior, etc. -->

MaryLJ committed:
updates for consistency with template, punctuation updated

MaryLJ committed:
updated for consistency with template

MaryLJ committed:
minor reference updates; move figure title below figure (for some); bold a figure title; e.g. and i.e. to e.g., and i.e.; add square brackets around xaction number; punctuation updates

MaryLJ committed:
capitalization updates

MaryLJ committed:
formatting of closed issues list updates

MaryLJ committed:
punctuation update

MaryLJ committed:
Update actor and transaction definitions based on discussion during DCC review/approval call.

kinsonho commented on input/pagecontent/volume-1.md:
```diff
@@ -983,7 +983,7 @@ The Hub shall support [content sharing](https://build.fhir.org/ig/HL7/fhircast-d
 The Hub shall monitor the established websocket connections. If it detects a websocket connection issue with a Subscriber, then the Hub shall:
 - Unsubscribe the Subscriber and drop the websocket connection
-- Send a SyncError notification to other Subscribers using [RAD-155](rad-155.html)
+- Send a SyncError notification to other Subscribers using \[RAD-155\](rad-155.html)
```
The proper change to include the [...] square bracket should be `\[[RAD-155](rad-155.html)\]` (without the double quotes). i.e. Add `\[` and `\]` at the beginning and end.

kinsonho committed:
Update sushi-config.yaml

Updated links to Actors and Transactions section to match the update in section title.

kinsonho commented on sushi-config.yaml:
```diff
@@ -116,20 +116,20 @@ pages:
   Volume 1:
     Introduction: volume-1.html
-    Actors and Transactions: volume-1.html#1531-integrating-reporting-applications
+    Actors and Transactions: volume-1.html#1531-ira-actors-transactions-and-content-modules
```
I made this change to correct the link that matches the change in the section title.

kinsonho committed:
Update volume-1.md

Correct the syntax to add square bracket around the transaction.

kinsonho commented on input/pagecontent/volume-1.md:
```diff
@@ -983,7 +983,7 @@ The Hub shall support [content sharing](https://build.fhir.org/ig/HL7/fhircast-d
 The Hub shall monitor the established websocket connections. If it detects a websocket connection issue with a Subscriber, then the Hub shall:
 - Unsubscribe the Subscriber and drop the websocket connection
-- Send a SyncError notification to other Subscribers using [RAD-155](rad-155.html)
+- Send a SyncError notification to other Subscribers using \[[RAD-155](rad-155.html)\]
```
I corrected the syntax here. RAD-155 should now still be a hyperlink.

kinsonho reviewed (approved):
Looks good. Thanks for the updates.

I made two fixes. One in Volume 1 line 986. One in sushi-config.yaml.

JohnMoehrke committed:
one more link needed fixing

JohnMoehrke committed:
move projected release to 10/4

kinsonho committed:
Corrected RAD-152 transaction name for Hub.

kinsonho committed:
Removed TODO and mark future work as a note
